coldfront/core/utils/templates/upload_mou.html

```diff
@@ -0,0 +1,40 @@
+{% extends "common/base.html" %}
+{% load crispy_forms_tags %}
+{% load humanize %}
+{% load static %}
+{% load feature_flags %}
+
+{% block title %}
+MOU Upload
```

Comment:
References to MOU and "Memorandum..." might need to be replaced with RUA and "Researcher..."?

`{{ mou_type_long }}` might already do this? Not sure.

Reply:
added a variable to keep track